website/docs/typescript-support.md

```diff
@@ -5,37 +5,22 @@ title: TypeScript Support
 
 ## Setup
 
-Docusaurus supports writing and using TypeScript theme components. To start using TypeScript, add `@docusaurus/module-type-aliases` to your project:
+Docusaurus supports writing and using TypeScript theme components. To start using TypeScript, add `@docusaurus/module-type-aliases` and some `@types` dependencies to your project:
 
 ```bash
-npm install --save-dev typescript @docusaurus/module-type-aliases
+npm install --save-dev typescript @docusaurus/module-type-aliases @types/react @types/react-router-dom @types/react-helmet
 ```
 
-Then add `tsconfig.json` to your project root with following content:
+Then add `tsconfig.json` to your project root with the following content:
 
 ```json title="tsconfig.json"
 {
-  "compilerOptions": {
-    "allowJs": true,
-    "esModuleInterop": true,
-    "jsx": "react",
-    "lib": ["DOM"],
-    "noEmit": true,
-    "noImplicitAny": false
-  },
+  "extends": "@tsconfig/docusaurus/tsconfig.json",
```

Comment:
Moves all these config options to https://github.com/tsconfig/bases/pull/27 - then it's obvious what is framework vs  project